### PR TITLE
fix: change locales, policies & initialData of the provider to required

### PR DESCRIPTION
**Pull request by ritazoliveira**
Changed the  `NextIntlProvider`'s `locales`, `policies` and `initialProps` props to required since they are indeed required further to use the `next-intl` library.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -12841,9 +12841,9 @@
       }
```
Comment (satazor) [inline package-lock.json]: Strange that we got changes to the `package-lock`. Have you removed it and installed the dependencies again?

**Comment by satazor** [comment]
Looks good, just a minor comment.

File: package-lock.json (hunk 1 above)
Comment (ritazoliveira) [reply package-lock.json]: Yes, there were some packages with vulnerabilities that were updated.

Comment (satazor) [reply package-lock.json]: Alright!

**Pull request merged**